### PR TITLE
Fix issue caused when font size calculated with long decimals

### PR DESCRIPTION
**Pull request by rolandkakonyi**
In my case the calculated value was `20.019...` which actually didn't display properly, it cut off the last line, but when rounded to have 1 decimal (`20.0`) it rendered with the full text.

My "workaround" might not be the most elegant, but does the job.

I'm open for suggestions to improve it.

### CODE REVIEW

File: Source/UILabelExtension.swift
Comment (tbaranes) [inline Source/UILabelExtension.swift:64]: The `rounded(.down)` isn't enough?

**Review by tbaranes — CHANGES_REQUESTED** [review]
Thanks for the fix, your case makes sense, round it is will be for the best.
can you also update the CHANGELOG?

File: Source/UILabelExtension.swift
Comment (rolandkakonyi) [reply Source/UILabelExtension.swift:64]: That was my initial solution, but then noticed the `0.1` diff threshold which would make no sense if we then round. So respecting that I rounded to keep 1 decimal.
If you say that having effectively Integer numbers as `calculatedFontSize` then `.rounded(.down)` would be enough.

About CHANGELOG:
What version should I put there, with what release date?
I think this is something that you as the maintainer should decide.

Comment (tbaranes) [reply Source/UILabelExtension.swift:64]: Got it, we can go with your solution then.
For the CHANGELOG, just add an entry below "bugfixes in the [CHANGELOG file](https://github.com/tbaranes/FittableFontLabel/blob/master/CHANGELOG.md)

Comment (rolandkakonyi) [reply Source/UILabelExtension.swift:64]: argh, sorry, completely missed the `## Next` section :D

Done.

**Review by tbaranes — APPROVED** [review]
All good, thanks!

**Pull request merged**
